AirCasting/Helpers/DeviceHandler.swift

```diff
@@ -0,0 +1,65 @@
+// Created by Lunar on 29/11/2021.
+//
+
+import Foundation
+import SwiftUI
+import DeviceKit
+
+struct DeviceHandler {
+    // Listed every device that support the app but has BT lover then 5.0
+    static let BTBelow5Devices: [Device] = [.iPhone4,
+                                            .iPhone5,
+                                            .iPhone5c,
+                                            .iPhone5s,
+                                            .iPhone6,
+                                            .iPhone6Plus,
+                                            .iPhone6s,
+                                            .iPhone6sPlus,
+                                            .iPhone7,
+                                            .iPhone7Plus,
+                                            .iPodTouch5,
+                                            .iPodTouch6,
+                                            .iPodTouch7,
+                                            .iPad2,
+                                            .iPad3,
+                                            .iPad4,
+                                            .iPad5,
+                                            .iPad6,
+                                            .iPad7,
+                                            .iPad8,
+                                            .iPad9,
+                                            .iPadAir,
+                                            .iPadAir2,
+                                            .iPadMini2,
+                                            .iPadMini3,
+                                            .iPadMini4,
+                                            .iPadPro9Inch,
+                                            .iPadPro10Inch,
+                                            .iPadPro12Inch,
+                                            .iPadPro12Inch2]
+    
+    static var disableButtonBasedOnBT: Bool {
+        DeviceHandler.BTBelow5Devices.contains(Device.current)
+    }
+    
+    static func notSupportedAlert() -> Alert {
+        Alert(
+            title: Text(Strings.DeviceHandler.alertTitle),
+            message: Text(Strings.DeviceHandler.alertMessage),
+            dismissButton: .default(Text(Strings.DeviceHandler.continueText))
+        )
+    }
+    
+    static func getDeviceNumber() -> Int {
```

Comment:
Is this function used anywhere?

Reply:
Not yer, but I wrote it and it could be useful so I would like it to stay :)